Documentation/devel/release.md

```diff
@@ -71,36 +70,37 @@ Now we switch to the GitHub web UI to conduct the release:
   This is a simple tarball:
 
 ```
-	export NAME="rkt-v1.2.0"
-	mkdir $NAME
-	cp release-build/bin/rkt release-build/bin/stage1-{coreos,kvm,fly}.aci $NAME/
-	cp -r dist/* $NAME/
-	sudo chown -R root:root $NAME/
-	tar czvf $NAME.tar.gz --numeric-owner $NAME/
+export RKTVER="1.2.0"
+export NAME="rkt-$RKTVER"
+mkdir $NAME
+cp release-build/bin/rkt release-build/bin/stage1-{coreos,kvm,fly}.aci $NAME/
+cp -r dist/* $NAME/
+sudo chown -R root:root $NAME/
+tar czvf $NAME.tar.gz --numeric-owner $NAME/
 ```
 
-- Attach the release signature; your personal GPG is okay for now:
+- Attach each stage1 file individually so they can be fetched by the ACI discovery mechanism. The files must be named as follows:
 
 ```
-	gpg --detach-sign $NAME.tar.gz
+cp release-build/bin/stage1-coreos.aci stage1-coreos-$RKTVER-linux-amd64.aci
+cp release-build/bin/stage1-kvm.aci stage1-kvm-$RKTVER-linux-amd64.aci
+cp release-build/bin/stage1-fly.aci stage1-fly-$RKTVER-linux-amd64.aci
 ```
 
-- Attach each stage1 file individually so they can be fetched by the ACI discovery mechanism. The files must be named as follows:
+- Sign all release artifacts.
 
-```
-	cp release-build/bin/stage1-coreos.aci stage1-coreos-1.2.0-linux-amd64.aci
-	cp release-build/bin/stage1-kvm.aci stage1-kvm-1.2.0-linux-amd64.aci
-	cp release-build/bin/stage1-fly.aci stage1-fly-1.2.0-linux-amd64.aci
-```
+rkt project key must be used to sign the generated binaries and images.`$RKTSUBKEYID` is the key ID of rkt project Yubikey. Connect the key and run `gpg2 --card-status` to get the ID.
```

Comment:
I stole this as-is from etcd guide, so I guess the article shouldn't be there :smile: